container/integration-test/Dockerfile

```diff
@@ -1,19 +1,22 @@
 ARG VERSION
 FROM gardenlinux/base-test:${VERSION}
 
-RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \
+RUN : "Install AWS requirements" \
+     && curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \
```

Comment:
I think we'll need a switch case here for `arm64` support. Next to this, I just saw that always a `x86_64` is taken which might fail when running on `arm64` based systems (like Apple Silicon).

Reply:
Good point. The main purpose of this PR is to get the `nightly` builds up an running again. Luckily, the corresponding Github Action that performs integration tests runs on an `amd64` machine so we do not have a direct need here.

However, since the Garden Linux build also supports `arm64` build systems, we should also ensure that base and integration tests can be run on `arm64` as well especially if someone already uses `arm64` for building Garden Linux (e.g by using Apple Silicon). I would say, let's create another issue for the `arm64` support and focus on fixing the nightly builds here.

Reply:
I'm fine with this. thanks